CONTRIBUTING.md

```diff
@@ -152,6 +152,21 @@ rustup override set master
 cargo test
 ```
 
+After fixing the build failure on this repository, we can submit a pull request
+to [`rust-lang/rust`] to fix the toolstate.
+
+To submit a pull request, you should follow these steps:
+
+```
+# Assuming you already cloned the rust-lang/rust repo and you're in the correct dir
+git submodule update --remote src/tools/clippy
```

Comment:
```suggestion
$ git submodule update --remote src/tools/clippy
```

Also in the commands below.

Reply:
We have no `$`s above snippet in the same section. Should we still have them?

Reply:
No in that case, just leave them out. Thanks for checking!